management/src/test/java/org/ehcache/management/providers/statistics/MissCountTest.java

```diff
@@ -0,0 +1,143 @@
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.terracotta.management.model.context.Context;
+
+/**
+ *
+ *
+ */
+@RunWith(Parameterized.class)
+public class MissCountTest {
+
+  @Rule
+  public final Timeout globalTimeout = Timeout.seconds(10);
+
+  @Rule
+  public final TemporaryFolder diskPath = new TemporaryFolder();
+
+  private final ResourcePools resources;
+  private final List<String> statNames;
+  private final List<Long> tierExpectedValues;
+  private final Long cacheExpectedValue;
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return asList(new Object[][] {
+    //1 tier
+    { newResourcePoolsBuilder().heap(1, MB), Arrays.asList("OnHeap:MissCount"), Arrays.asList(2L), 2L },
+    { newResourcePoolsBuilder().offheap(1, MB), Arrays.asList("OffHeap:MissCount"), Arrays.asList(2L), 2L },
+    { newResourcePoolsBuilder().disk(1, MB), Arrays.asList("Disk:MissCount"), Arrays.asList(2L), 2L },
+
+    //2 tiers
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB), Arrays.asList("OnHeap:MissCount","OffHeap:MissCount"), Arrays.asList(2L,2L), 2L},
+    { newResourcePoolsBuilder().heap(1, MB).disk(2, MB), Arrays.asList("OnHeap:MissCount","Disk:MissCount"), Arrays.asList(2L,2L), 2L},
+    //offheap and disk configuration below is not valid.  Throws IllegalStateException no Store.Provider found to handle configured resource types [offheap,disk]
+    
+    //3 tiers
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB).disk(3, MB), Arrays.asList("OnHeap:MissCount","OffHeap:MissCount","Disk:MissCount"), Arrays.asList(2L,2L,2L), 2L}
+    });
+  }
+
+  public MissCountTest(Builder<? extends ResourcePools> resources, List<String> statNames, List<Long> tierExpectedValues, Long cacheExpectedValue) {
+    this.resources = resources.build();
+    this.statNames = statNames;
+    this.tierExpectedValues = tierExpectedValues;
+    this.cacheExpectedValue = cacheExpectedValue;
+  }
+
+  @Test
+  public void test() throws InterruptedException, IOException {
+    DefaultManagementRegistryConfiguration registryConfiguration = new DefaultManagementRegistryConfiguration().setCacheManagerAlias("myCacheManager");
+    registryConfiguration.addConfiguration(new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES));
+    ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(registryConfiguration);
+
+    Configuration cacheConfiguration = new DefaultConfiguration(EvictionTest.class.getClassLoader(),
+            new DefaultPersistenceConfiguration(diskPath.newFolder()));
+
+    Collection<Service> services = new ArrayList<Service>();
+    services.add(managementRegistry);
+
+    CacheManager cacheManager = null;
+
+    try {
+      cacheManager = new EhcacheManager(cacheConfiguration, services);
```

Comment:
You are right, that would make more sense since the management tests should use the api. I will change it.